### PR TITLE
[3454][ADD] purchase_reception_status

### PR DESCRIPTION
**Pull request by rinaldifirdaus**
[3454](https://www.quartile.co/web#id=3454&action=771&model=project.task&view_type=form&menu_id=505)

### CODE REVIEW

**Comment by rinaldifirdaus** [comment]
This module is extended from the original OCA Repo by adjust receipt_status to be invisible.
will update this module with the original one once its being merged and added the adjustment to hid the receipt_status.

File: purchase_reception_status/views/purchase_order.xml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        </field>
+    </record>
+    <record id="purchase_order_view_form" model="ir.ui.view">
+        <field name="name">received_status.purchase.order.view.form</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase_stock.purchase_order_view_form_inherit" />
+        <field name="arch" type="xml">            
+            <xpath expr="//field[@name='receipt_status']" position="attributes">
+                <attribute name="invisible">True</attribute>
+            </xpath>
```
Comment (yostashiro) [inline purchase_reception_status/views/purchase_order.xml]: Move this to above record and remove this record.

**Review by yostashiro — CHANGES_REQUESTED** [review]
Please add another commit with your changes on top of the OCA PR content, so that it will be easy to identify your changes later.

**Comment by yostashiro** [comment]
> Please add another commit with your changes on top of the OCA PR content, so that it will be easy to identify your changes later.

@rinaldifirdaus Commit history currently doesn't reflect my request.  The commits we should have here are:

1. A commit that mirrors the content of the OCA PR.
2. Another commit with your changes.

**Comment by rinaldifirdaus** [comment]
> @rinaldifirdaus Commit history currently doesn't reflect my request. The commits we should have here are:
> 1. A commit that mirrors the content of the OCA PR.
> 2. Another commit with your changes.

@yostashiro san, I've just added the new commits as above suggestions.
kindly your review. thank you

**Review by yostashiro — APPROVED** [review]

**Comment by rinaldifirdaus** [comment]
Commit squashed

**Pull request merged**